src/components/Settings/Organization/AccountLists/AccountListRow/AccountListRow.test.tsx

```diff
@@ -97,42 +124,103 @@ describe('AccountLists', () => {
   });
 
   describe('Handling Deletions', () => {
+    it('should delete an accountList', async () => {
+      const { getByRole, getByText, getByTestId } = render(
+        <Components accountList={accountList} search="" organizationId="" />,
+      );
+      const reason = 'Because I am an admin';
+
+      userEvent.hover(getByTestId('DeleteAccountListButton'));
+      await waitFor(() => {
+        expect(getByText('Permanently delete this account.')).toBeVisible();
+      });
+
+      userEvent.click(getByRole('button', { name: 'Delete Account' }));
+      const reasonTextField = getByRole('textbox', { name: 'Reason' });
+      userEvent.type(reasonTextField, reason);
+      userEvent.click(getByRole('button', { name: 'Yes' }));
+
+      await waitFor(() => {
+        expect(mutationSpy).toHaveGraphqlOperation('DeleteAccountList', {
+          input: { accountListId: '1111', reason },
+        });
+
+        expect(mockEnqueue).toHaveBeenCalledWith(
+          'Deletion process enqueued: Name1',
+          {
+            variant: 'success',
+          },
+        );
+      });
+    });
+
+    it('should not show delete icon if the account list is in more than 1 organization', async () => {
+      const alteredAccountList = { ...accountList };
+      alteredAccountList.organizationCount = 2;
+      const { queryByTestId } = render(
+        <SnackbarProvider>
+          <TestRouter router={router}>
+            <GqlMockedProvider onCall={mutationSpy}>
+              <ThemeProvider theme={theme}>
+                <AccountListRow
+                  accountList={alteredAccountList}
+                  search=""
+                  organizationId=""
+                />
+              </ThemeProvider>
+            </GqlMockedProvider>
+          </TestRouter>
+        </SnackbarProvider>,
+      );
+
+      expect(queryByTestId('DeleteAccountListButton')).not.toBeInTheDocument();
```

Comment:
Since you have to wait for the query to return, I wonder if you should wrap this in a waitFor or wait for something that shows GraphQL has finished fetching.

This isn;t super important, but making sure we're not passing a test which always passes.